eZ/Publish/API/Repository/Values/Content/Content.php

```diff
@@ -28,6 +28,18 @@
     abstract public function getVersionInfo();
 
     /**
+     * Shorthand method for getVersionInfo()->getName().
+     *
+     * @see \eZ\Publish\API\Repository\Values\Content\VersionInfo::getName()
+     *
+     * @param string|null $languageCode
+     *
+     * @return string|null The name for a given language, or null if $languageCode is not set
+     *         or does not exist.
+     */
+    abstract public function getName($languageCode = null);
```

Comment:
> An abstract is a BC break. Making it concrete would prevent the break.

> However as before when it comes to API BC promise, in #2002 didn't really consider implementations of API our BC promise, but rather consumption.

We've also had a brief discussion about it in the office with @Nattfarinn and @lserwatka last week and conclusion on that was the same - we promise API BC on consumption and don't recommend extending it.
Extendable "things" should be in SPI, right?